deploy/images/README.md

```diff
@@ -14,7 +14,12 @@ The result is an image, named `egarchive/lega-base`, and containing `python 3.6`
 
 ## Dependencies
 
-`rabbitmq:3.6.14-management`, `postgres:9.6`, `python:3.6-alpine3.8` are pulled from the main Docker hub.
+The following images are pulled from the main Docker hub:
+
+* `egarchive/lega-inbox` (based on OpenSSH version 7.8p1 and CentOS7)
+* `egarchive/lega-mq` (based on `rabbitmq:3.6.14-management`)
```

Comment:
It will be great to add link of the Git repos to these images, i.e. `https://github.com/EGA-archive/LocalEGA-mq`  and  `https://github.com/EGA-archive/LocalEGA-db`

Reply:
They are linked in the paragraph that follows.

Reply:
not for `lega-mq` and `lega-db`